composeApp/src/commonMain/kotlin/com/github/familyvault/ui/screens/main/familyGroupSettings/familyGroupMember/ModifyFamilyMemberScreen.kt

```diff
@@ -134,6 +143,10 @@ class ModifyFamilyMemberScreen(private val familyMember: FamilyMember) : Screen
                                 familyMember.fullname,
                                 selectedPermissionGroup
                             )
+                            val updatedUser = familyGroupService.retrieveFamilyGroupMembersList().single { it.publicKey == familyMember.publicKey }
+                            if (updatedUser.permissionGroup == selectedPermissionGroup) {
```

Comment:
Tutaj nawet jak Guardian zostanie Guardianem to też będziemy aktualizowali wszystkie wątki chatu? Może lepiej będzie sprawdzić czy została tutaj grupa uprawnień zmieniona.